VideoLocker/res/layout/row_course_outline_list.xml

```diff
@@ -120,14 +119,11 @@
 
             <org.edx.mobile.view.custom.IconImageViewXml
                 android:id="@+id/bulk_download"
-                android:layout_width="18dp"
-                android:layout_height="18dp"
+                android:layout_width="34dp"
+                android:layout_height="48dp"
                 android:baselineAlignBottom="true"
-                android:paddingEnd="1dp"
-                android:paddingLeft="0dp"
-                android:paddingRight="1dp"
-                android:paddingStart="0dp"
-                android:paddingTop="3dp"
+                android:paddingStart="4dp"
+                android:paddingEnd="14dp"
```

Comment:
paddingStart/paddingEnd weren't added until API 17; you need to keep paddingLeft and paddingRight as fallbacks.